src/AnnouncementBubble/NormalAnnouncementBubble.tsx

```diff
@@ -115,6 +127,35 @@ function formReplyButton(
   return undefined;
 }
 
+function formReadReceiptsTooltip(
+  readBy: string[],
+  recipientType: "student" | "guardian",
+): JSX.Element {
+  const readReceiptCount = readBy.length;
+  const readReceiptString = convertReadReceiptArrayToString(readBy);
+  const recipientString = readBy.length === 1 ? `${recipientType}` : `${recipientType}s`;
```

Comment:
In the first case, I dont think you need the `{}` since we just set the string = to recipientType